.github/workflows/toolshed.yaml

```diff
@@ -19,24 +19,11 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        include:
-        - test-install-client: 'galaxy_api'
-          python-version: '3.7'
-          shed-api: 'v1'
-          shed-browser: 'twill'
-        - test-install-client: 'standalone'
-          python-version: '3.8'
-          shed-api: 'v1'
-          shed-browser: 'twill'
-        - test-install-client: 'galaxy_api'
-          python-version: '3.9'
-          shed-api: 'v2'
-          shed-browser: 'playwright'
-        - test-install-client: 'standalone'
-          python-version: '3.10'
-          shed-api: 'v2'
-          shed-browser: 'playwright'
```

Comment:
I was "attempting" to only testing a subset of configurations because I thought all against all on the these combinations would be too much testing for such a niche feature. Does this configuration double the number of tool shed tests we're running?

Reply:
Yes, from 4 to 8. If you think that some of these combinations are not worth to be run on both Python versions, I can redo the matrix to run them only on Python 3.7.